src/shell/server.cpp

```diff
@@ -387,7 +387,7 @@ void server::run() {
        We added this hack because the json library code used for ensuring this property
        was crashing when compiling Lean on Windows with mingw. */
 #if !defined(LEAN_EMSCRIPTEN)
-    std::setlocale(LC_NUMERIC, "C");
+    setlocale(LC_NUMERIC, "C");
 #endif
```

Comment:
Sorry not to be clear: my compiler was complaining about this with the `std::`. I reported the error message at https://github.com/leanprover/lean/issues/1481, but closed it soon after as the merge of Gabriel's parallel compilation branch made the problem go away.